spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe "the contestants index page" do
+  #user story 2
+  it "shows a list of names of all the contestants and their projects" do
+      
+    recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+    news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    boardfit = recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+    upholstery_tux = furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    lit_fit = furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+    jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+
+    ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+    ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id)
+    ContestantProject.create(contestant_id: gretchen.id, project_id: upholstery_tux.id)
+    ContestantProject.create(contestant_id: kentaro.id, project_id: upholstery_tux.id)
+    ContestantProject.create(contestant_id: kentaro.id, project_id: boardfit.id)
+    ContestantProject.create(contestant_id: erin.id, project_id: boardfit.id)
+    
+    visit "/contestants"
+
+    expect(page).to have_content(jay.name)
+    expect(page).to have_content(gretchen.name)
+    expect(page).to have_content(news_chic.name)
```

Comment:
Your expect statements here are fine, but there's not enough of them to really satisfy the requirements you wrote in the "it" block above: 
```
shows a list of names of all the contestants and their projects
```

I'm only seeing tests for the names of the contestants and one project, not for all of them. News Chic does belong to both of them, but Upholstery Tux belongs to just Gretchen. 
I think a better way to write this test would be to use a `within` block and make sure that only Jay's information is in Jay's area, and vice-versa with Gretchen's. Try to refactor this using a `within` block for good practice!